upgrade/1.2/scripts/upgrade/prerequisites.sh

```diff
@@ -361,7 +370,9 @@ else
     echo "====> ${state_name} has been completed"
 fi
 
-# Take cps deployment snapshot
-cps_deployment_snapshot=$(cray cps deployment list --format json | jq -r '.[] | .node' || true)
-echo $cps_deployment_snapshot > /etc/cray/upgrade/csm/${CSM_RELEASE}/cp.deployment.snapshot
-
+# Take cps deployment snapshot (if cps installed)
+kubectl get pod -n services | grep -q cray-cps
```

Comment:
we have set -e in prereq so this will exit the script and print out a failure message i think

Reply:
great catch!  fixed..